src/Commands/Admin.py

```diff
@@ -45,6 +47,49 @@ def brokenQueue(roles: List[Role], mentions: List[Member]) -> Embed:
         )
 
 
+async def forceReport(mentions: List[Member], roles: List[Role], lbChannel = channel, *arg) -> Embed:
```

Comment:
Make sure we're typing lbChannel to be of type Channel instead of setting the default value to Channel.

I believe this should be `async def forceReport(..., lbChannel: Channel, *arg) -> Embed:`